projects/storefrontlib/src/cms-components/product/product-list/container/product-list.model.ts

```diff
@@ -0,0 +1,45 @@
+/**
```

Comment:
Should this file be exported to the public API?

Reply:
blush, yes.